sinks_test.go

```diff
@@ -166,6 +172,30 @@ func TestSHA256Sum(t *testing.T) {
 	}
 }
 
+func TestSinkSlice(t *testing.T) {
+	t.Parallel()
+
+	want := []string {
+		"testdata/multiple_files/1.txt",
+		"testdata/multiple_files/2.txt",
+		"testdata/multiple_files/3.tar.zip",
+	}
+	got, err := ListFiles("testdata/multiple_files").Slice()
```

Comment:
Aren't we testing ListFiles here too, which we don't need to do? I would use a test case of a string containing embedded newlines, and test that we get them correctly as a slice.